app/src/features/projects/view/ProjectPage.tsx

```diff
@@ -30,6 +31,10 @@ const ProjectPage: React.FC = () => {
   const [isLoadingCodes, setIsLoadingCodes] = useState(false);
   const [codes, setCodes] = useState<IGetAllCodeSetsResponse>();
 
+  const projectContext = useContext(ProjectContext);
+
+  console.log({ projectContext });
```

Comment:
nit

Reply:
Intentionally left this in to facillitate testing. Could remove it now... or after it gets an approval?